Main/Peripherals/include/Peripherals/ThrustController.h

```diff
@@ -37,8 +39,11 @@ class ThrustController
 
     FloatPair forwardTrim;
     FloatPair diveTrim;
+    FloatPair diveOffset;
 
     FloatPair getReciprocalValues(float value);
+    float getSafeOffset(float a, float b);
+    void zeroPowerHelper(float &a, float &b);
```

Comment:
Setting the sub's dive to `0` and setting each of the dive thrusters to `0` are not necessarily the same thing.
We would want to preserve the trimming and offsets, even when dive is set close to zero so that the sub remains level as it's surfacing.
This line is basically saying, "If either dive thruster is set to zero, they should both be set to zero." I don't feel like that's an assumption that the `setThrust()` function should necessarily be making.